master/paths.html

```diff
@@ -231,6 +231,10 @@ <h3 id="TheDProperty">Specifying path data: the <span class='property'>'d'</span
 If the path data string contains no valid commands, then the behavior
 is the same as the <span class='prop-value'>none</span> value.</p>
 
+<p>When <a>'d'</a> is parsed as an attribute, it is parsed according to the
```

Comment:
Could you be more specific what this means for the path() function please? I thing what you want to say is that the grammar would not allow path() and falls back to EBNF as specified in this spec.